opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -71,22 +71,19 @@ namespace Opm {
                        const GridProperties<int>* intGridProperties,
                        const GridProperties<double>* doubleGridProperties)
         {
-            const auto iter = std::find_if( values.begin(), values.end(), []( double value ) { return std::isnan(value); });
-            if (iter != values.end()) {
-                const auto& poro = doubleGridProperties->getKeyword("PORO");
-                const auto& ntg =  doubleGridProperties->getKeyword("NTG");
-
-                if (poro.containsNaN())
-                    throw std::logic_error("Do not have information for the PORV keyword - some defaulted values in PORO");
-                else {
-                    const auto& poroData = poro.getData();
-                    for (size_t globalIndex = 0; globalIndex < poro.getCartesianSize(); globalIndex++) {
-                        if (std::isnan(values[globalIndex])) {
-                            double cell_poro = poroData[globalIndex];
-                            double cell_ntg = ntg.iget(globalIndex);
-                            double cell_volume = eclipseGrid->getCellVolume(globalIndex);
-                            values[globalIndex] = cell_poro * cell_volume * cell_ntg;
-                        }
+            const auto& poro = doubleGridProperties->getKeyword("PORO");
+            const auto& ntg =  doubleGridProperties->getKeyword("NTG");
+
+            if (poro.containsNaN())
```

Comment:
I see: You're right, but I suppose that PORO only needs to be defined for cells where PORV is NaN? that case is not handled correctly by the current code either, right?

anyway: fix pushed, thanks.